cmd/provisioner-localpv/app/provisioner.go

```diff
@@ -131,7 +134,7 @@ func (p *Provisioner) Delete(pv *v1.PersistentVolume) (err error) {
 	defer func() {
 		err = errors.Wrapf(err, "failed to delete volume %v", pv.Name)
 	}()
-
+	glog.Infof("%+v", pv)
```

Comment:
We need to clean up in actual PR

Reply:
I need to get it *dummy* working in my minikube, so I am still working on it. Currently this part of code isn't getting hit in my minikube `v1.14`.

Reply:
Now removed